plugins/@grouparoo/mongo/src/lib/table-import/connection.ts

```diff
@@ -15,16 +15,6 @@ export function getConnection() {
       "Import or update Profiles from a Mongo DB database collection.",
     tableOptionDescription: "The table to scan",
     tableOptionDisplayName: "Collection",
-    sourceOptions: {
-      options: [
-        {
-          key: "fields",
-          displayName: "Fields",
-          required: true,
-          description: "Comma separated list of fields",
-        },
-      ],
-    },
```

Comment:
Since the mongo plugin is so new, I think it's OK to not worry about what a migration looks like here.  

* In the future, we would need to remove old options in the database or else the app is no longer valid
* We would also need to instruct people how to change the code-config they have